bleak/backends/winrt/client.py

```diff
@@ -113,10 +113,14 @@ class BleakClientWinRT(BaseBleakClient):
     a package that enables Python developers to access Windows Runtime APIs directly from Python.
 
     Args:
-        address_or_ble_device (`BLEDevice` or str): The Bluetooth address of the BLE peripheral to connect to or the `BLEDevice` object representing it.
+        address_or_ble_device (``BLEDevice`` or str): The Bluetooth address of the BLE peripheral
```

Comment:
```suggestion
        address_or_ble_device (str or BLEDevice): The Bluetooth address of the BLE peripheral
```

Would be nice to put type in same order as descriptions. Also, no backticks should be necessary here.

Reply:
Hadn't noticed that. You are very right.

Will Google-style do string args render differently than other text? Will check it out and change.

Reply:
It should pick up the type and automatically create a link.